apps/demo/package.json

```diff
@@ -15,8 +15,8 @@
     "@remix-run/node": "^1.5.1",
     "@remix-run/react": "^1.5.1",
     "@remix-run/serve": "^1.5.1",
-    "react": "18.1.0",
-    "react-dom": "18.1.0"
+    "react": "^18.2.0",
```

Comment:
While removing `react` from the core component dev dependencies work, it's easier to just upgrade `react` in the Remix app to the latest.